stdlib/2/typing.pyi

```diff
@@ -61,47 +62,60 @@ _V_co = TypeVar('_V_co', covariant=True)  # Any type covariant containers.
 _KT_co = TypeVar('_KT_co', covariant=True)  # Key type covariant containers.
 _VT_co = TypeVar('_VT_co', covariant=True)  # Value type covariant containers.
 _T_contra = TypeVar('_T_contra', contravariant=True)  # Ditto contravariant.
+_TC = TypeVar('_TC', bound=Type[object])
 
-class SupportsInt(metaclass=ABCMeta):
+def runtime(cls: _TC) -> _TC: ...
+
+@runtime
```

Comment:
Since this is a stub, why does it need `@runtime`?

Reply:
Otherwise mypy will complain about e.g. ``issubclass(it, collections.abc.Iterable)`` something like ``isinstance() can be used only with @runtime protocols``.